### ISSUE TITLE
[err] Did not receive response to shouldStartLoad in time, defaulting to YES

### ISSUE THREAD

**Issue by byein** [BaedalMate/BaedalMate_iOS#33]
- [ ] fix this issue

https://github.com/react-native-webview/react-native-webview/issues/124

**Comment by Attique033** [comment]
@byein any luck with this?

**Comment by byein** [comment]
@Attique033 Yeah. After following the guide from https://github.com/react-native-webview/react-native-webview/issues/124, there's no more error about this, but I need to check in real iphone for testing. Because it is related to the current location(GPS).

**Comment by Attique033** [comment]
@byein I've tried adding the `onShouldStartLoadWithRequest={() => true}` but my case is still same, can you share the fix here
